src/pages/docs/fundamentals.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import { PrismJSON } from '../../components/Prism'
+import simpleWebExample from '../../examples/simple-web.json'
+import Layout from '../../components/layout'
+import ClassHierarchy from '../../components/ClassHierarchy'
+
+class FundamentalsPage extends React.Component {
+  render() {
+    return (
+      <Layout title="Fundamentals">
+        <h1>AXRL Fundamentals</h1>
+        <p>
+          Accessibility Reporting Language (AXRL) is a data
+          format designed to describe accessibility tests
+          for Web pages, Native apps and other software.
+          AXRL is designed to be a flexible and extendable
```

Comment:
I think the work you need here is "extensible": "designed to allow the addition of new capabilities and functionality" instead of "extendable": "able to be made longer or larger"